Observability.md

```diff
@@ -0,0 +1,122 @@
+
+Spans also have more advanced concepts like a status to indicate if the span completed successfully. It is also possible to link spans together. Spans can also have events attached to them to indicate point in time events similar to how log events are captured.
+
+While OpenTelemetry uses terms like `spans`, `baggage`, `traces`, `metrics`, not all of the libraries that support OpenTelemetry collectors use the same terms.  For example, in .NET span is represented by an Activity in the DiagnosticSource library. The OpenTelemetry ecosystem is a bit messy in its design because it is result of a set of pragmatic compromises between a large number of existing APM (Application Performance Monitoring) vendors that already had existing solutions.  The benefit is broad operability and future that is far more aligned than the past.
+
+The important take-away from this for us as SDK developers, is that with this slightly different model for capturing information about our running SDKs, we open up a wide range of possibilities for developers writing apps with our SDKs to get very clear visibility into what is happening inside our library.
+
+## What to capture - tracing
+
+Open Telemetry spans are nested and can have attributes attached to each span. The following spans should be created for each request with the indicated attributes.
+
+```JSONC
+{
+  "ApplicationRequest?": { // this span is owned by the application itself in the calling code that calls the chained API surface or equivalent
+    "SetContentFromParsable?" { // request information or equivalent when a request body is used from a model, also applies to setContentFromScalar and from stream
+      "com.microsoft.kiota.request.type": "Fully qualified name of the content model"
+    },
+    "SendAsync - <URI template>": { // http request adapter or equivalent, also applies to the other overloads like sendPrimitiveAsync etc...
+      "http.response_content_length?": "int value for the response content length",
+      "http.status_code": "http response status code",
+      "http.flavor": "1.1 or 2.0",
+      "http.response_content_type?": "value from the response content type header",
```

Comment:
maybe this should now be
`http.response.body.content_type`

Reply:
created a separate issue https://github.com/microsoftgraph/msgraph-sdk-design/issues/99